deployment/network-operator/templates/upgrade-crd.yaml

```diff
@@ -76,5 +76,8 @@ spec:
         {{- if .Values.sriovNetworkOperator.enabled }}
               kubectl apply -f /crds/sriov-network-operator;
         {{- end }}
+        {{- if .Values.nfd.enabled }}
+              if ! kubectl get crd | grep -E "nodefeaturerules\.nfd\.k8s-sigs\.io|nodefeatures\.nfd\.k8s-sigs\.io" ; then kubectl apply -f /crds/node-feature-discovery; fi
```

Comment:
if nfd is enabled, i think we always want to update its CRDs. (like we did above)

e.g when we version bump NFD and a new field is added to CRD but version remains the same.